miden/tests/integration/operations/stack_ops.rs

```diff
@@ -1,6 +1,6 @@
 use crate::build_op_test;
+use proptest::prelude::*;
 use rand_utils::rand_vector;
-
```

Comment:
@grjte Cargo fix removes the extra line.

Reply:
Interesting - that's not happening for me, and I think it must not be happening for Bobbin either, since most of our files are formatted with a blank line there. It's not a big deal though & not worth you fighting with rustfmt, so let's leave it like this

Reply:
Ah so sorry, it's not. I thought you meant 2 blank lines. There's no problem with 1 blank line. My bad

Reply:
Oh cool, no problem!